raylib/platforms/rcore_android.c

```diff
@@ -129,7 +129,7 @@ void android_main(struct android_app *app)
 }
 
 // NOTE: Add this to header (if apps really need it)
-struct android_app *GetAndroidApp(void)
+extern struct android_app *GetAndroidApp(void)
```

Comment:
reason for this is in pure C we can make `#include "core_android.h"`, but in go would be pain ...